cmd/check_tx_lookup.cpp

```diff
@@ -131,15 +117,15 @@ int main(int argc, char* argv[]) {
                 SILKWORM_LOG(LogLevel::Info) << "Scanned blocks " << expected_block_number << std::endl;
             }
 
-            if (should_stop_) {
+            if (SignalHandler::signalled()) {
```

Comment:
should be `signaled` with one `l`

Reply:
signalled is also correct: https://www.merriam-webster.com/dictionary/signal

Reply:
you are right, my bad. It is less common (about 10 times fewer results on google), but yes!